Nick/69 (#74)
tlswg/tls-subcerts: draft-ietf-tls-subcerts.md

traffic, while using RSA certificates without the DelegationUsage certificate
extension for non-DC traffic; this completely prevents such attacks.

Note that if a signature can be forged over an arbitrary credential, the
attacker can choose any value for the valid_time field.  Repeated signature
forgeries therefore allow the attacker to create multiple delegated
credentials that can cover the entire validity period of the

> certificate.  It is therefore safe to assume that temporary exposure

Reviewer: might I suggest a different wording here:

s/It is therefore safe to assume that/Temporary exposure of the key or a signing oracle, may allow the attacker ....

The it is safe to assume makes it look like it is safe, but maybe its actually dangerous.

> of the key or a signing oracle implies that the attacker may be able to

Reviewer 2: ```suggestion
of the key or a signing oracle implies that the attacker can
```
